README.md

```diff
@@ -69,18 +70,18 @@ var body: some View {
 }
 ```
 
-And just like that, you've got a workflow! You can now add more items to it or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
+And just like that, you've got a workflow! You can now add more items to it or reorder the items that are there. To understand more of how this works, [check out our developer docs.](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Check out our example apps
-We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
+# Try Our Example Apps
```

Comment:
Does this change the meaning at all? It was more like "I want you to look at the code that backs our example apps and understand what it is doing"

Whereas now it reads to me more like "I want you to go run our example apps"

Reply:
Ah, I see. I changed it because we say "check out" in the previous sentence.

Reply:
How about Look at Our Example Apps?

Reply:
```suggestion
# Look at Our Example Apps
```
How does this sound, @Tyler-Keith-Thompson?